# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../towitoko/00-reader-conf-lib-path.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -ur towitoko-2.0.7/doc/reader.conf patched-towitoko-2.0.7/doc/reader.conf
--- towitoko-2.0.7/doc/reader.conf	2001-10-12 18:52:31.000000000 +0200
+++ patched-towitoko-2.0.7/doc/reader.conf	2003-02-23 14:39:47.000000000 +0100
@@ -12,7 +12,7 @@
 # Towitoko Chipdrive Micro (COM1)
 FRIENDLYNAME	"Towitoko Chipdrive Micro"
 DEVICENAME	TOWITOKO_CHIPDRIVE_MICRO
-LIBPATH		/usr/local/towitoko/lib/libtowitoko.so.2.0.0
+LIBPATH		/usr/lib/libtowitoko.so.2.0.0
 CHANNELID	0x000001
 
 # Note: Use the full path on LIBPATH.
